lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -136,12 +136,21 @@ public:
     SetSessionStatus(const uwb::protocol::fira::UwbSessionStatus& status);
 
     /**
-     * @brief Get the application configuration parameters for this session.
+     * @brief Get the Application Configuration Parameters object
      *
+     * @param requestedTypes leave this as an empty vector to request all parameters
      * @return std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
      */
     std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
-    GetApplicationConfigurationParameters();
+    GetApplicationConfigurationParameters(std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> requestedTypes);
+
+    /**
+     * @brief Set the Application Configuration Parameters object
```

Comment:
Let's update this text to match the text for the `Get...*` call above (after restoring its old text and mentioned in separate comment).